test-code/demo_for_oi_active_cycle.R

```diff
@@ -0,0 +1,121 @@
+library(dplyr)
```

Comment:
Great to see this demo code, really useful! I should be added to .Rbuildignore if not already, there is a function in usethis for that I think.

Reply:
[`use_build_ignore`](https://usethis.r-lib.org/reference/use_build_ignore.html) it seems! 

There is alredy this within .Rbuildignore: `^test-code$` - I'm not sure if this will ignore everything within the `test-code` directory, which includes this demo script so I have left for now (assuming this is indeed the case).

Reply:
That should ignore everything in the folder so all good.